packages/builder/src/builderStore/store/index.js

```diff
@@ -190,6 +193,58 @@ const createScreen = store => async screen => {
   await savePromise
 }
 
+const createLink = store => async (url, title) => {
+  let savePromise
+  store.update(state => {
+    // Try to extract a nav component from the master screen
+    const nav = findChildComponentType(
+      state.pages.main,
+      "@budibase/standard-components/Navigation"
+    )
+    if (nav) {
+      let newLink
+
+      // Clone an existing link if one exists
+      if (nav._children && nav._children.length) {
```

Comment:
This is really cool - if you pre-styled your links it will respect that when the auto ones are generated. 👌